CHANGELOG.md

```diff
@@ -3,20 +3,13 @@ All notable changes to this project are documented in this file.
 
 Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.1.0/)
 
-## [Unreleased 3.x](https://github.com/opensearch-project/flow-framework/compare/3.0...HEAD)
+## [Unreleased 3.1](https://github.com/opensearch-project/flow-framework/compare/3.0...HEAD)
```

Comment:
Don't we keep this as 3.x and pick up the changes to a release note files separately?

Reply:
The `x` or `1` is just a label. The important part is what we are comparing against on the link, which needs to refer to the latest release (currently 3.0).

We will have to change this line anyway when we reset the changelog after a release (after 3.1.0 we will change the link to reference `3.1...HEAD`) so I don't think it matters much whether we put x or 1.  I'm happy to put this back if there's a strong opinion to do so (how are other plugins doing it?)

Reply:
Kept it as 3.x in neural search https://github.com/opensearch-project/neural-search/blob/main/CHANGELOG.md?plain=1#L6